handlers/load_test_handler.go

```diff
@@ -61,7 +69,7 @@ func (h *Handler) LoadTestUsingSMPHandler(w http.ResponseWriter, req *http.Reque
 		return
 	}
 	// meshName := q.Get("mesh")
-	testUUID := perfTest.Id
+	profileID := perfTest.Id
```

Comment:
This is good :+1:. I think users should provide configuration for a profile. The one which contains profile ID.
Test ID isn't something that they should give but should be generated internally. So we might not want to leave `testUUID` empty here or else it will automatically get the profile name.

Reply:
I'll add the logic to generate a test ID

Reply:
We can leave test-id to be empty as test-id will be automatically generated and will be attached to the test-result.